internal/job/id.go

```diff
@@ -0,0 +1,15 @@
+package job
+
+import "go.jetify.com/typeid"
+
+type Prefix struct{}
+
+func (Prefix) Prefix() string { return "job" }
```

Comment:
Is it required to add a prefix? Aren't generally available UUIDs good enough?


also add comments/godoc

Reply:
I've found it really helpful when dealing with large systems, that may have lots of different kinds of identifiers, to ensure that they are prefixed. When digging through logs, it may not be clear what a UUID is, but prefixed IDs stand out. Also, these IDs are K-sortable and based on the upcoming UUID-7 standard, so they're not _that_ different.